docker-compose/compose.yaml

```diff
@@ -268,9 +339,17 @@ services:
       - "--entryPoints.traefik.address=:8888"
       - "--accesslog=true"
       - "--providers.docker.exposedByDefault=false"
+      - "--ping.entrypoint=web"
```

Comment:
```suggestion
```

<img width="356" alt="Screenshot 2024-10-21 at 17 59 04" src="https://github.com/user-attachments/assets/ab4a02d2-8701-421d-8e5a-56c9c84e0b60">

maybe it's not really needed, looks like just a name for the entrypoint in the config
https://doc.traefik.io/traefik/operations/ping/#entrypoint

Reply:
This file removed or set to traefik doesnt work. Can find any documentation why this happens